cppapi/client/group.h

```diff
@@ -157,11 +157,17 @@ class GroupReply
               bool group_element_enabled = true);
   //- dtor
   virtual ~GroupReply ();
-  //- group_element_enabled accessor
+///@publicsection
+/**
+ * Check if the group element coresponding to this reply is enabled.
```

Comment:
coresponding => corresponding (lines 162 and 164)

Reply:
Not sure it is necessary to duplicate the line. I understand one is a summary and the other should be a more detailed explanation on what this method is doing. In this case, it is redundant since the summary is detailed enough...?
I don't mind keeping it like that but I find it cleaner to avoid duplication.

Reply:
Both short and long description are shown when you click the function name in the html documentation, so I removed the long one. Also, we have other functions with only the short description, so the change seems to be ok. Thanks!